.travis.yml

```diff
@@ -17,12 +17,19 @@ install:
   - make install-ci
 
 services:
+  - docker
   - postgresql
   - redis
 
 before_script:
+  - docker run -d -p 8000:8000 dwmkerr/dynamodb
+  - sleep 1  # let DynamoDB start
```

Comment:
speed in travis is unpredictable, there's no guarantee 1s would be enough.

can we instead use some simple health check script?

Reply:
I've added a simple health check.